exist-ant/src/main/java/org/exist/ant/ListGroupsTask.java

```diff
@@ -39,57 +39,83 @@ public class ListGroupsTask extends UserTask
     /* (non-Javadoc)
```

Comment:
WHen adding (new) javadoc, please remove the old (non java)doc, and add the previous @See to the new text.